### DISCUSSION TITLE
Ligero y poderoso tooltip con CSS

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#3] [Announcements]
# Ligero y poderoso tooltip con CSS

Hoy te comparto un proyecto más en el cual llevo trabajando bastante tiempo, mis propios globos informativos que no requieren javascript

https://www.zkreations.com/2022/09/tooltips-pure-css-nojs.html

**Comment by eibelfiles** [comment]
Excelente, y hermosa nueva página web saludos!

**Reply by danieIabel** [reply]
Muchas gracias por pasar y comentar amigo. Lo unico que si me pone triste es la copia de seguridad de Disqus.</3

**Comment by ghost** [comment]
Está quedando genial!!

**Reply by danieIabel** [reply]
Muchas gracias por pasar y comentar amigo! Cada comentario nuevo me ayuda a pasar el trago amargo de los que no pude migrar </3

**Reply by davidlimonche** [reply]
jejeje.. acabo de ver la documentación de como responder con imagines en giscus.. y la verdad es que es un dolor de cabeza :/ 

tienes que postear una imagen con url y no con un botón ;(

![This is an image](https://images.emojiterra.com/google/android-11/512px/1f625.png)

**Reply by danieIabel** [reply]
La forma en la que funcionan los comentarios en Giscus no es propia amigo. Ese lenguaje se conoce como markdown, por ejemplo, para poner un enlace seria:

```markdown
[Nombre enlace](http://www.google.com)
```

Para una imagen es igual pero con el símbolo `!` delante:

```markdown
![nombre imagen](http://www.example.com/image.jpg)
```

markdown es muy conocido, aprenderlo te ayudará a trabajar mejor en tus proyectos, sobre todo cuando trates de documentarlos. Puedes ver todo sobre markdown aquí: [Markdown Guide](https://www.markdownguide.org/basic-syntax/)

**Comment by Jousethlc** [comment]
Volviste, que bien :D 
psdt: está muy bueno el tooltip

**Reply by danieIabel** [reply]
Muchas gracias por pasar y comentar amigo. Esta nueva versión aprovecha lo mas reciente en CSS y está construida con el fin de ser lo más pequeña posible. Seguiré actualizando este proyecto siempre que encuentre mejoras aplicables.

**Reply by Jousethlc** [reply]
Esta muy bueno, Gracias :D
(Estoy esperando esa nueva versión de la plantilla "Anime" ♥)

**Comment by TakoMics** [comment]
Este tutorial me recuerda aquellos del año 2014, pero claro, ahora con esteroides y actualizado.

Gracias Daniel.

¿Piensas actualizar el YouTube Lazy Load? Recuerdo que se llamaba "YouTube ítems". Igual entiendo que vas poco a poco renovando contenido, sea como sea, estamos contentos de tenerte de vuelta.

**Reply by danieIabel** [reply]
Claro amigo, estoy considerando prácticamente todos los articulos (digo prácticamente ya que algunos los voy a fusionar porque son parecidos). Si intentas acceder a los enlaces notaras que no están borrados por completo, solo deje un aviso de actualización.

**Comment by Framberdev** [comment]
Gracias por estupendo proyecto. Es un gran plus para Blogger. Bienvenido de vuelta.

Saludos amigo. 🙂

**Reply by danieIabel** [reply]
Muchas gracias por el recibimiento amigo!. Un saludo.

**Comment by joelcordoba** [comment]
Hola, Daniel me gusta mucho el nuevo diseño de la página, una pregunta va a hacer algún tutorial de como agregar esta caja de comentarios a Blogger.

**Reply by danieIabel** [reply]
Buenas noches amigo, si desea usar el mismo sistema, bastaría con seguir los pasos oficiales: [giscus.](https://giscus.app/es) aunque si se complica mucho y hay más personas interesadas, en ese caso realizaré un tutorial.

**Comment by Ursso** [comment]
Espectacular, sin palabras me he quedado. Saludos
